great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -374,36 +383,474 @@ def apply_theme(
         theme = DataAssistantResult._get_theme(theme=theme)
         return [chart.configure(**theme) for chart in charts]
 
+    @staticmethod
+    def get_nominal_metric_chart(
+        df: pd.DataFrame,
+        metric_name: str,
+        sequential: bool,
+        subtitle: Optional[str] = None,
+    ) -> alt.Chart:
+        """
+        Args:
+            df: A pandas dataframe containing the data to be plotted
+            metric_name: The name of the metric as it exists in the pandas dataframe
+            sequential: Whether batches are sequential in nature
+            subtitle: The subtitle, if applicable
+
+        Returns:
+            An altair chart for nominal metrics
+        """
+        batch_name: str = "batch"
+        batch_identifiers: List[str] = [
+            column for column in df.columns if column not in [metric_name, batch_name]
+        ]
+        batch_type: alt.StandardType
+        if sequential:
+            batch_type = AltairDataTypes.ORDINAL.value
+        else:
+            batch_type = AltairDataTypes.NOMINAL.value
+        batch_component: BatchPlotComponent = BatchPlotComponent(
+            name=batch_name,
+            alt_type=batch_type,
+            batch_identifiers=batch_identifiers,
+        )
+
+        column_number: str = "column_number"
+
+        metric_type: alt.StandardType = AltairDataTypes.NOMINAL.value
```

Comment:
Similar comment around naming - how about `metric_plot_type`?

Reply:
I'm not sure if `metric_plot_type` works because there can be a type for each visualized dimension in the plot. I think it is pretty accurate to call this the `metric_type` and if we add this information to the metric itself, it would be even better.

Reply:
Makes sense - so the `metric_type` on the metric would come from a GE enum and then translate to the appropriate Altair enum here (just so we can isolate the plotting library and swap it out in the future without breaking changes if we need to).